torch_geometric/sampler/neighbor_sampler.py

```diff
@@ -119,7 +119,10 @@ def __init__(
             feature_store, graph_store = data
 
             # Obtain graph metadata:
-            node_attrs = feature_store.get_all_tensor_attrs()
+            node_attrs = [
```

Comment:
@rusty1s  Thank you for the review :). One comment about these lines: they are common for both homo and hetero. In case of homo the attr.group_name will be None and therefore the condition `isinstance(attr.group_name, NodeType)` will never be met and node_attrs will be always empty. I will open a new PR with the previous version.